terraform/github-wif.tf

```
@@ -0,0 +1,92 @@
+# Copyright 2023 The Authors (see AUTHORS file)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+locals {
+  repo_full_name = "${var.wif_github.owner_name}/${var.wif_github.repo_name}"
+
+  default_wif_attribute_mapping = {
+    "google.subject" : "assertion.sub"
+    "attribute.actor" : "assertion.actor"
+    "attribute.aud" : "assertion.aud"
+    "attribute.event_name" : "assertion.event_name"
+    "attribute.environment" : "assertion.environment"
+    "attribute.repository" : "assertion.repository"
+    "attribute.repository_id" : "assertion.repository_id"
+    "attribute.repository_owner_id" : "assertion.repository_owner_id"
+    "attribute.repository_visibility" : "assertion.repository_visibility"
+    "attribute.workflow" : "assertion.workflow"
+    "attribute.workflow_ref" : "assertion.workflow_ref"
```

We can remove the unnecessary values from this:
 
```suggestion
    "google.subject" : "assertion.sub"
    "attribute.actor" : "assertion.actor"
    "attribute.aud" : "assertion.aud"
    "attribute.repository_owner_id" : "assertion.repository_owner_id"
```